src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2362,7 +2362,7 @@ private void CloseHomeWorkspace(object parameter)
 
         private bool CanCloseHomeWorkspace(object parameter)
         {
-            return HomeSpace.RunSettings.RunEnabled;
+            return HomeSpace.RunSettings.RunEnabled || parameter is true;
```

Comment:
hmm I guess this is for enabling closing the workspace when trust warning is shown?

Reply:
When a graph takes a considerable amount of time, do we want allow closing the workspace?

Reply:
I think the real issue here is that `RunEnabled` was only meant to block certain actions to protect VM but is now also used for trust warning. Is it possible to still block it when computation is running but enable when trust warning is displayed?

Reply:
Removed this additional check as well.